frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -869,7 +881,7 @@ function VirtualKeyboard:addKeys()
     local keyboard_frame = FrameContainer:new{
         margin = 0,
         bordersize = Size.border.default,
-        background = Blitbuffer.COLOR_WHITE,
+        background = G_reader_settings:readSetting("night_mode") and Blitbuffer.COLOR_WHITE or Blitbuffer.COLOR_LIGHT_GRAY,
```

Comment:
Nope.

Which device was this tested on?

Reply:
Okay, actually saw your edit in the discussion, so, this may actually make sense, but I'd still like an answer to the platform question, because this might not be the best way to go at this ;).

Reply:
I tested this on a Kobo, not sure why that's related. I think I see what you're getting at though, having the keys as dark gray makes the whole thing run a looot slower so I'll probably fix that.

Reply:
I was actually thinking you might have prototyped this on the emu, where night mode and color constants may be surprising in some edge cases, but, yeah, gray blobs are usually a bad idea on eInk ;).